### ISSUE TITLE
Conflict when loading multiple sGDML models

### ISSUE THREAD

**Issue by fonsecag** [stefanch/sGDML#4]
**Issue**: Loading multiple sGDML models (using GDMLPredict) in the same script seems to create conflicts, making you unable to use previously loaded models. The most recently loaded model works fine. 

**Version**: sgdml 0.3.2 [python 3.6.9]

**OS**: Mac OS 10.14.6 

**Steps**: Can easily be reproduced on my machine using the following script (here using uracil and toluene as an example)
```python
import numpy as np
from sgdml.predict import GDMLPredict

uracil_data= np.load('datasets/uracil.npz')
X_uracil=np.reshape(uracil_data['R'],(len(uracil_data['R']),-1))
toluene_data=np.load('datasets/toluene.npz')
X_toluene=np.reshape(toluene_data['R'],(len(toluene_data['R']),-1))

uracil_npz=np.load('models/uracil.npz')
uracil_model=GDMLPredict(uracil_npz)

E,_=uracil_model.predict(X_uracil[0])
print(f"E={E}")

toluene_npz=np.load('models/toluene.npz')
toluene_model=GDMLPredict(toluene_npz)

#leads to error:
#ValueError: operands could not be broadcast together with remapped shapes [original->remapped]: (66,) and requested shape (12000,105)
E,_=uracil_model.predict(X_uracil[0]) 
```

**Comment by stefanch** [comment]
Thank you very much for pointing out that issue and for providing a nice example to reproduce it! We've identified the culprit and are working on a fix.

**Comment by stefanch** [comment]
Fixed in version 0.4.3
